### DISCUSSION TITLE
Is there a way to set up a default readme for a GitHub organization?

### DISCUSSION THREAD

**Discussion by resthomas** [community/community#22806] [New to GitHub]
<p>Within a team or organization is it possible to supply a default readme that, when a new repository is created, will automatically appear and have questions/fields for the creator to fill out?</p>
<p>Cheers</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/resthomas">@resthomas</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>To address your question, it sounds like you’d like to have a README template (similar to an issue template) so that any new repo that gets opened has the same standard formatting and content across all repos within a given organization. I do not think this is currently possible; however, I can understand how that could be valuable.</p>
<p>We’re always working to improve GitHub, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>
<p>Cheers!</p>

**Comment by resthomas** [comment]
<p>Thanks Nadia!</p>

**Comment by amarnathrayudu** [comment]
<p>Hi Team,<br>
Any update on adding this provision? Readme option at organization level is really helpful.</p>
<p>Thanks,</p>
<p>Amarnath</p>

**Comment by PetrKryslUCSD** [comment]
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a> I would like to have a README for the organization itself, not for the repos.<br>
Explain what the org is about, what resources are available, etc. Is <em>that</em> possible?</p>

**Reply by Grizzlly** [reply]
That is already possible.
